dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_tool.jsp

```diff
@@ -311,3 +343,31 @@
 
     </form>
 </div>
+
+<div dojoType="dijit.Dialog" id="deleteBundleActions" title='<%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, "bundle.delete.title" )) %>'>
+    <table class="sTypeTable">
+        <tr>
+            <td id="selectedBundlesBtnContainer">
+                <button id="deleteSelectedBundles" dojoType="dijit.form.Button" class="dijitButton" onClick="deleteSelectedAudits()">
+                    <%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, "bundle.delete.selected")) %>
+                </button>
+            </td>
+            <td>
+                <button id="deleteALLBundles" dojoType="dijit.form.Button" class="dijitButton" onClick="deleteAllAudits()">
+                    <%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, "bundle.delete.all")) %>
+                </button>
+            </td>
+            <td>
+                <button id="deleteSuccessBundles" dojoType="dijit.form.Button" class="dijitButton" onClick="deleteSuccessAudits()">
+                    <%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, "bundle.delete.success")) %>
+                </button>
+            </td>
+            <td>
+                <button id="deleteFailBundles" dojoType="dijit.form.Button" class="dijitButton" onClick="deleteFailAudits()">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)